sound/soc/sof/core.c

```diff
@@ -408,7 +424,7 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 
 	if (IS_ERR(plat_data->pdev_mach)) {
 		ret = PTR_ERR(plat_data->pdev_mach);
-		goto fw_run_err;
+		goto register_err;
```

Comment:
The goto target is in conditionally compiled code ?
```
#if !IS_ENABLED(CONFIG_SND_SOC_SOF_PROBE_WORK_QUEUE)
register_err:
```

Reply:
good catch @lgirdwood yes, we should add an empty one to the other condition case.